src/components/BillStatusTag/BillStatusTag.svelte

```diff
@@ -0,0 +1,28 @@
+<script lang="ts">
+	import { BillStatus } from '$models/bill';
+	import { Tag } from 'carbon-components-svelte';
+
+	export let isLarge = false;
+	export let status: BillStatus;
+	let label = status;
```

Comment:
Assigning one variable directly to another one is not necessary, you can just use `{status}` in the HTML template.